aten/src/ATen/core/Dimname_test.cpp

```diff
@@ -0,0 +1,57 @@
+#ifdef NAMEDTENSOR_ENABLED
```

Comment:
do we ever build this?

Reply:
Yes, it always gets built

Reply:
so what happens if you try to run this, then?

Reply:
if NAMEDTENSOR_ENABLED, then
```
Running main() from ../third_party/googletest/googletest/src/gtest_main.cc
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from DimnameTest
[ RUN      ] DimnameTest.isValidIdentifier
[       OK ] DimnameTest.isValidIdentifier (0 ms)
[ RUN      ] DimnameTest.wildcardName
[       OK ] DimnameTest.wildcardName (0 ms)
[ RUN      ] DimnameTest.createNormalName
[       OK ] DimnameTest.createNormalName (2 ms)
[ RUN      ] DimnameTest.createTaggedName
[       OK ] DimnameTest.createTaggedName (3 ms)
[ RUN      ] DimnameTest.unifyAndMatch
[       OK ] DimnameTest.unifyAndMatch (0 ms)
[----------] 5 tests from DimnameTest (5 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (5 ms total)
[  PASSED  ] 5 tests.
```
otherwise there are 0 tests.

Reply:
I should figure out how to hook this up to the CI, I'm not sure this is being run right now